### PR TITLE
Modify styling for listview items

### LINKED ISSUE #30: Extend width of row highlight to left side
Currently the left padding of the listView makes it so the row highlight does not go all the way to the left, which looks strange.

I played a bit around with giving each item of the list left padding, like with a loop as mentioned [on StackOverflow](https://stackoverflow.com/a/12250639) but it does not work:

Any idea how we can achieve that?

### LINKED ISSUE #31: Change highlight color on long-press from orange
The orange color on long-press is very old-Androidy and doesn’t fit the style of the app. It should simply be white or grey.

### LINKED ISSUE #42: Orange highlight on press or long-press
When you press or long-press an entry, it gets an orange highlight background. This doesn’t really fit the style of Light Launcher.

Better would be to either just use white background – or maybe just underline it (like light phone does) as that’s not such a harsh change.

light launcher underline

This article might be helpful: [ListView Tips & Tricks #3: Create Fancy ListViews](https://cyrilmottier.com/2011/08/08/listview-tips-tricks-3-create-fancy-listviews/)

### PR DESCRIPTION
**Pull request by compost-em-all**
Resolves #42 
Also resolves #31 which sounds like a duplicate Issue of #42
Should also resolve #30 since I had to move the creation of the
left-padding down the actual listview items to get onClick effect to
extend across the whole width of the screen.

### CODE REVIEW

**Comment by compost-em-all** [comment]
example

File: LaLauncher/src/main/java/com/github/postapczuk/lalauncher/AppsActivity.java [hunk 1]
```diff
@@ -90,6 +101,14 @@ public void onClickHandler() {
     @TargetApi(Build.VERSION_CODES.GINGERBREAD)
     public void onLongPressHandler() {
         listView.setOnItemLongClickListener((parent, view, position, id) -> {
+            TextView selectedItem = view.findViewById(view.getId());
```
Comment (postapczuk) [inline LaLauncher/src/main/java/com/github/postapczuk/lalauncher/AppsActivity.java]: duplicated code - you could extract it to method in AppsActivity, and use it in both places

File: LaLauncher/src/main/java/com/github/postapczuk/lalauncher/MainActivity.java [hunk 2]
```diff
@@ -108,6 +111,14 @@ public void fetchAppList() {
     @Override
     public void onClickHandler() {
         listView.setOnItemClickListener((parent, view, position, id) -> {
+            TextView selectedItem = view.findViewById(view.getId());
```
Comment (postapczuk) [inline LaLauncher/src/main/java/com/github/postapczuk/lalauncher/MainActivity.java]: Ok, now it's trippled - as I mentioned create method in AppsActivity accessible for all classes that extend it

File: LaLauncher/src/main/java/com/github/postapczuk/lalauncher/AllAppsActivity.java [hunk 3]
```diff
@@ -42,4 +60,11 @@ public void onSwipeBottom() {
             }
         });
     }
+
+    // Set the left padding at the item level vs the listview level
+    private void applyItemPadding(TextView item){
```
Comment (postapczuk) [inline LaLauncher/src/main/java/com/github/postapczuk/lalauncher/AllAppsActivity.java]: duplicated private method (here and in MainActivity). Move it from both (AllAppsActivity and MainActivity) to the AppsActivity with proper access (remove `private` word)

**Review by postapczuk — CHANGES_REQUESTED** [review]

**Comment by jancborchardt** [comment]
Very very nice @taylor-josh! :) How much opacity is it? It looks like it could be even more transparent as it’s very harsh white / light grey?

**Comment by compost-em-all** [comment]
Thanks @jancborchardt!

For the highlight effect behind the items when they are clicked? That is
set at 40% transparency (60% opacity). I was attempting to get the colors close to what was
already being used. When I was testing, it didn't seem that harsh to me.
Thoughts, @postapczuk ?

**Review by postapczuk — APPROVED** [review]

**Comment by jancborchardt** [comment]
@taylor-josh right, 60% opacity – for the mockup at https://github.com/light-launcher/Light-Android-Launcher/pull/54#issuecomment-472625092 it is 4% opacity for example, and still very visible. :)

What do you think about using something lower like 4%, or 10%?

**Comment by compost-em-all** [comment]
Yeah, I can always take a look at dropping the opacity some the next time I am in the project working on something.

**Comment by jancborchardt** [comment]
@taylor-josh or let me know which values I need to modify? :)

**Comment by compost-em-all** [comment]
The background color should be in the colors.xml file and I believe is
called colorBackgroundFavorite. Now that I think about it, that is a
crappy name since it's not only used in the favorite screen.

**Pull request merged**